connector-sdk/jackson-datatype-document/src/main/java/io/camunda/connector/document/jackson/OperationModel.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.document.jackson;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public record OperationModel(@JsonProperty(DISCRIMINATOR_KEY) String name, List<Object> params) {
+
+  public static final String DISCRIMINATOR_KEY = "camunda.operation.type";
```

Comment:
Just food for thought, we could also add the name to the property like this: 
```camunda.operation::base64``` and we would have an object that is slightly more dense:
```
{"camunda.operation::base64": [...]``

Reply:
Agree. Consistency is key.